glide-core/src/client/value_conversion.rs

```diff
@@ -243,6 +224,15 @@ pub(crate) fn expected_type_for_cmd(cmd: &Cmd) -> Option<ExpectedReturnType> {
         b"ZSCORE" => Some(ExpectedReturnType::DoubleOrNull),
         b"ZPOPMIN" | b"ZPOPMAX" => Some(ExpectedReturnType::MapOfStringToDouble),
         b"JSON.TOGGLE" => Some(ExpectedReturnType::JsonToggleReturnType),
+        b"ZADD" => cmd
+            .position(b"INCR")
```

Comment:
what if incr is in lower case?

Reply:
won't work. you'll need to change CMD in order to make this work.

Reply:
or, alternatively, add another allocation to this function.

Reply:
I was wrong. works out of the box.
added a commit with tests.